ospf6d/ospf6_lsdb.c

```diff
@@ -91,6 +91,72 @@ static void _lsdb_count_assert(struct ospf6_lsdb *lsdb)
 #define ospf6_lsdb_count_assert(t) ((void) 0)
 #endif /*DEBUG*/
 
+/* Increment LSA-Type Count in Databases LSA Statistics */
+inline void ospf6_lsdb_stats_inc(struct ospf6_lsa *lsa, struct ospf6_lsdb *lsdb)
+{
+	switch (ntohs(lsa->header->type)) {
+	case OSPF6_LSTYPE_ROUTER:
```

Comment:
Food for thought - it's possible to avoid copying this switch block into the `dec` function below if both `inc` and `dec` called into a common one that took a bool or somesuch determining whether to add `-1` or `1` to each value here.

Not necessary but it'd reduce the probability of copy paste errors in case this code is ever changed.

Reply:
fixed.